### DISCUSSION TITLE
关于“自动导出”功能

### DISCUSSION THREAD

**Discussion by jia6y** [jia6y/flomo-to-obsidian#26] [Announcements]
目前“自动导出”功能还需依赖playwright，所以使用该功能需要预先安装playwright到本地。

**Comment by aplus42** [comment]
<img width="595" alt="iShot_2024-04-07_11 04 30" src="https://github.com/jia6y/flomo-to-obsidian/assets/1201865/3ab4879c-44b3-4146-8c55-6b7da465b924">
执行安装命令后报错，应该在什么目录下安装？

**Reply by jia6y** [reply]
嗯 **npx playwright install**

**Reply by aplus42** [reply]
<img width="538" alt="iShot_2024-04-07_13 28 41" src="https://github.com/jia6y/flomo-to-obsidian/assets/1201865/91d91305-ab12-4306-a6fc-94ec51a96687">
<img width="346" alt="iShot_2024-04-07_13 22 15" src="https://github.com/jia6y/flomo-to-obsidian/assets/1201865/02291d6a-2003-44a5-837a-fb159e487098">
安装了npx playwright install，还是报错

**Reply by aplus42** [reply]
<img width="486" alt="iShot_2024-04-07_13 31 33" src="https://github.com/jia6y/flomo-to-obsidian/assets/1201865/7e602220-2363-445a-944c-ee510ed65f8a">
没有1983，有1982

**Reply by jia6y** [reply]
我换了个driver，卸载flomo importer插件然后再安装试试？

**Reply by aplus42** [reply]
卸载重装还是不行啊，目录当中有chromium-1112目录，但取的是chromium-1105目录，这个是不是写的有问题啊

**Reply by aplus42** [reply]
最后通过改文件路径的方式解决了

**Reply by hnzyc** [reply]
哈哈，重装多次无果，我也通过修改文件路径方式实现了

**Reply by jia6y** [reply]
抱歉，解决了，卸载插件重新转一下就好了。我之前用的playwright版本比较老。

**Reply by Paper-blue** [reply]
我也是修改路径解决的，哈哈

**Reply by wzm1241** [reply]
> 我也是修改路径解决的，哈哈

咋解决路径的

**Reply by Paper-blue** [reply]
> > 我也是修改路径解决的，哈哈
> 
> 咋解决路径的

修改chromium那个文件夹后面的数字

**Comment by jia6y** [comment]
请问一下Ob的版本是多少？这个问题看起来很奇怪，这些都不该是插件正确的行为

**Comment by jia6y** [comment]
请问undefined文件夹是我的插件生成的吗？

**Comment by wzm1241** [comment]
> 目前“自动导出”功能还需依赖playwright，所以使用该功能需要预先安装playwright到本地。

这个玩意咋搞。。。
